src/utils/csv-parser-allowed-channels.js

```diff
@@ -0,0 +1,16 @@
+function parseAllowedChannels(channels) {
+  if (!channels) {
+    return [];
+  }
+
+  return channels
+    .split(';')
+    .filter(Boolean)
+    .map((channelData) => {
+      const [id, name] = channelData.split(',');
+
+      return { id, name };
```

Comment:
This file and [this other](https://github.com/TeamNovaSoft/discord-bot/blob/db5efc411e5db378934d126be63234e6182553e9/src/csv-parser-allowed-channels.js) are the same, please remove one of them

Reply:
Completed